app/controllers/revisions_controller.rb

```diff
@@ -11,6 +11,7 @@ def index
   end
 
   def show
+    @class_name = @record.class.name.downcase
```

Comment:
it seems to me you don't need this variable.

You can replace the view "if else if else" with a case structure:

```ruby
<% case @record %>
<% when Card %>
<% when Issue %>
<% end %>